apps/token-manager/app/src/components/Panels/AssignVotePanelContent.js

```diff
@@ -115,6 +115,8 @@ class AssignVotePanelContent extends React.Component {
               value={amount}
               onChange={this.handleAmountChange}
               min={minTokenStep}
+              {...(maxAmount === '-1' ? {} : { max: maxAmount })}
```

Comment:
As a guard against weird calculated states, it might be worth it to also pass in `maxAccountTokens` here or use it earlier when we set the config, to make sure it's `Math.max(maxAmount, maxAccountTokens)`.

Reply:
Actually, I'm thinking we should move the `maxAmount` calculation completely into the panel, and just give the panel the holder's balance and the Token Manager's `maxAccountTokens`.

Right now, if you open the assign token panel without using the context menu, the max amount doesn't get computed correctly (`getHolderBalance()` always returns `-1`).